JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -2174,7 +2122,8 @@
         iter              = options$iteration,
         warmup            = options$warmup,
         adapt_delta       = options$adapt_delta,
-        control           = list(max_treedepth = options$max_treedepth)
+        control           = list(max_treedepth = options$max_treedepth),
+        seed              = if (isTRUE(options[["setSeed"]])) set.seed(options[["seed"]])
```

Comment:
these and others can just be:
```suggestion
        seed              = .setSeedJASP(options)
```
we've abstracted this into a function to avoid people from having to do this manually and so we can easily change the implementation of the seed relatively easily.

Reply:
but the `.setSeedJASP` function set's the seed in R and doesn't return the seed if it's specified.:

```
> JASP:::.setSeedJASP
function (options) 
{
    if (is.list(options) && c("setSeed", "seed") %in% 
        names(options)) {
        if (isTRUE(options[["setSeed"]])) 
            set.seed(options[["seed"]])
    }
}
```

Here, I need to pass it into the rstan function which handles seeds differently.

Reply:
but you had
```r
seed              = if (isTRUE(options[["setSeed"]])) set.seed(options[["seed"]])
```
shouldn't it then be
```r
seed              = if (isTRUE(options[["setSeed"]])) options[["seed"]]
```
without calling `set.seed` (which returns NULL)?

Reply:
yes, it definitely should've. Thanks for noticing this.

Reply:
interesting thing is that both of the ways worked for making the results reproducible

Reply:
perhaps it defaults to `.Random.seed` when the seed is NULL?